platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -2047,9 +2047,15 @@
             var exists;
 
             if ($scope.advancedSettings !== null && $scope.lookup !== undefined && $scope.lookup.lookupName !== undefined) {
-                LoadingModal.open();
                 $scope.validateLookupName($scope.lookup.lookupName);
-                LoadingModal.close();
+            }
+        });
+
+
+        $rootScope.$on('$stateChangeStart', function(event, toState, toParams, fromState, fromParams) {
```

Comment:
@wstrzelczyk @MichalKus We need to catch the change of state from `mds.schemaEditor` when `#advancedObjectSettingsModal` is open to force it to be closed on such state change. I can see that we catch the change of state only from `mds.schemaEditor` with either modal being open or closed